server/controllers/journal/fiscal.js

```diff
@@ -50,11 +51,11 @@ function close(id, user_id, data, cb) {
   function init() {
     cfg.user_id = user_id;
     cfg.project_id = 1; // HBB by default
-    transactionDate = cfg.isForClosing ? forcingDate : new Date();
-    return [
+    transactionDate = cfg.isForClosing ? util.toMysqlDate(forcingDate) : util.toMysqlDate();
+    return q.when([
```

Comment:
Do you really need q.when()?  [According to the Q documentation](https://github.com/kriskowal/q#combination), Q will create a new promise (using Q.all) internally, so you do not need to create one here.